packages/react-wildcat/src/staticServer.js

```diff
@@ -213,9 +215,11 @@ function start() {
 }
 
 function close() {
-    if (server && server.close) {
-        return server.close();
+    if (!server || !server.close) {
```

Comment:
Why the change from the && to the  ||? What case is this catching that the previous method wasn't? It seemed like the previous check with && was a lot nicer